src/Umbraco.Core/Migrations/Upgrade/V_8_0_0/DropDownPropertyEditorsMigration.cs

```diff
@@ -0,0 +1,167 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using Umbraco.Core.Persistence;
+using Umbraco.Core.Persistence.Dtos;
+using Umbraco.Core.PropertyEditors;
+using Umbraco.Core.Logging;
+
+namespace Umbraco.Core.Migrations.Upgrade.V_8_0_0
+{
+    public class DropDownPropertyEditorsMigration : MigrationBase
+    {
+        public DropDownPropertyEditorsMigration(IMigrationContext context) : base(context)
+        {
+        }
+
+        public override void Migrate()
+        {
+            //need to convert the old drop down data types to use the new one
+            var oldDropDowns = Database.Fetch<DataTypeDto>(Sql()
+                .Select<DataTypeDto>()
+                .From<DataTypeDto>()
+                .Where<DataTypeDto>(x => x.EditorAlias.Contains(".DropDown")));
+            foreach (var dd in oldDropDowns)
+            {
+                //nothing to change if there is no config
+                if (dd.Configuration.IsNullOrWhiteSpace()) continue;
+
+                ValueListConfiguration config;
+                try
+                {
+                    config = JsonConvert.DeserializeObject<ValueListConfiguration>(dd.Configuration);
+                }
+                catch (Exception ex)
+                {
+                    Logger.Error<DropDownPropertyEditorsMigration>(ex, $"Invalid drop down configuration detected: \"{dd.Configuration}\", cannot convert editor, values will be cleared");
```

Comment:
Try not to do string.interpolation for Serilog log messages

```suggestion
                    Logger.Error<DropDownPropertyEditorsMigration>(ex, "Invalid drop down configuration detected: \"{DropDownConfiguration}\", cannot convert editor, values will be cleared", dd.Configuration);
```